### PR TITLE
Feat/like counter

### PR DESCRIPTION
**Pull request by karljerico**
### OVERVIEW

- Added Like counter using AJAX 

### SNIPPETS

image
image
image

### CODE REVIEW

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

File: src/resources/views/post/index.blade.php [hunk 1]
```diff
@@ -21,10 +21,13 @@
                 @if ($post->image)
                     <img height="250px" width="150px" src="{{ asset('/storage/images/'.$post->image) }}" alt="Post Image" class="img-fluid">
                 @endif
+                <span class="like-count" data-post-id="{{ $post->id }}">
+                    {{ $post->likes->count() === 0 ? '' : $post->likes->count() . ' ' . ($post->likes->count() === 1 ? 'Like' : 'Likes') }}
```
Comment (hipe-francis) [inline src/resources/views/post/index.blade.php:25]: For future references: You can move this to your controller. Try to minimize logic handling in your blade files.

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

**Pull request merged**